src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1291,9 +1291,16 @@ smart_objects::SmartObjectSPtr MessageHelper::CreateAppVrHelp(
     return NULL;
   }
   smart_objects::SmartObject& vr_help = *result;
-  vr_help[strings::vr_help_title] = app->name();
-
+  vr_help[strings::vr_help_title] = 
```

Comment:
@ValeriiMalkov please add if condition and check:
1. Returned from `vr_help_title()` pointer is not NULL
2. `strings::vr_help_title` key exists in returned smart object

Reply:
@AKalinich-Luxoft That was fixed at the commit 8b18082df6387812f7a92870da98a12c9db9b15e